packages/acceptance-tests/pkg-tests-specs/sources/commands/publish.test.ts

```diff
@@ -53,7 +53,7 @@ describe(`publish`, () =>   {
         env: {
           YARN_NPM_AUTH_TOKEN: validLogins.otpUser.npmAuthToken,
         },
-      })).rejects.toThrow();
+      })).rejects.toThrowErrorMatchingInlineSnapshot();
```

Comment:
I know that this will fail right now - but I have problems running the test suite locally so I plan to check CI results and fix this then